tests/unit/sftp_test.py

```diff
@@ -48,6 +50,23 @@ def dst_path(file_paths_pair):
     return path
 
 
+@pytest.fixture(params=[32, 1025])
+def sized_payload():
+    """Generate a binary test payload of size given as a parameter."""
+    payload_len = 1024 + 1
```

Comment:
It's incomplete. 1024 is still hardcoded. It needs to be retrieved from the `request` fixture. Additionally, it's a good idea to always set IDs so that the semantic meaning would be clear.
```suggestion
@pytest.fixture(
    params=(32, 1024 + 1),
    ids=('small-payload', 'large-payload'),
)
def sized_payload(request: pytest.FixtureRequest) -> None:
    """Generate a binary test payload of size given as a parameter."""
    payload_len = request.param
```

Reply:
Oh, my bad. Missed that part. Thanks for guidance!